subsys/net/ip/Kconfig.ipv4

```diff
@@ -42,6 +42,14 @@ config NET_IPV4_DEFAULT_NETMASK
 
 if NET_NATIVE_IPV4
 
+config NET_IPV4_MTU
```

Comment:
please add to release notes so that we don't forget later :)

Reply:
Done